packages/backend-core/src/logging/alerts.ts

```diff
@@ -0,0 +1,26 @@
+const NonErrors = ["AccountError"]
+
+function isSuppressed(e?: any) {
+  return e && e["suppressAlert"]
+}
+
+export function logAlert(message: string, e?: any) {
```

Comment:
Can these `e` be typed?

Reply:
We tend to use `any` for errors, but I suppose we could use `Error` from here on out 👍